tests/c_languages/AStyleLintBearTest.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+from bears.c_languages.AStyleLintBear import AStyleLintBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+test_file1 = """
```

Comment:
could you create more descriptive variable names; instead of `test_file1`, maybe `test_good_file_with_assignment`